02-running-grakn/01-install-and-run.md

```diff
@@ -24,6 +24,67 @@ Download the [latest release](https://grakn.ai/download?os=mac_os_x#core), unzip
 Having downloaded Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
 [tab:end]
 
+[tab:CentOS]
+
+### To download the latest release
+
+As a superuser, add the repo:
+```
+# yum-config-manager --add-repo http://repo.grakn.ai/repository/meta/grakn-core.repo
```

Comment:
should it be `http` or `https`?

Reply:
@lolski it should be `https` here

Reply:
@lolski fixed in https://github.com/graknlabs/docs/pull/90/commits/1afb04207d4d2bfd9a78541517e773096e895647